README.md

```diff
@@ -27,6 +27,37 @@ Deck is a kanban style organization tool aimed at personal planning and project
 - [mail2deck](https://github.com/newroco/mail2deck) - Provides an "email in" solution 
 - [A-deck](https://github.com/leoossa/A-deck) - Chrome Extension that allows to create new card in selected stack based on current tab
  
+### Shortcuts
+
+Deck supports keyboard shortcuts to make working easier and faster for power users.
+This is the default layout:
+
+**Card Shortcuts**
+
+_These work when you hover over a card:_
+
+Key | Action
+---|---
+`spacebar` | un-/assigns yourself
+`enter` | open card
+`d` | set due date
+`a`,`m` | un-/assign any member
+`c` | archive card
+`l` | add tags/labels to card
+`t` | quick edit card title
+
+**Board Shortcuts**
+
+_These work when the board page is active:_
+
+Key | Action
+---|---
+`n` | create new card
+`/` | focus on search field
+`q` | only show your cards
+`f` | open filter menu
+`x` | clear all filters
+
```

Comment:
There is an open server issue about aligning keyboard shortcuts across apps. https://github.com/nextcloud/server/issues/1444 There doesn't seem to be too much of a consensus, though. Generally, I like adding a `?` shortcut to explain the shortcuts inside the app.